src/app/shared/contact/contact.component.less

```diff
@@ -0,0 +1,20 @@
+@primary-red-color: #F40009;
+
+#contact-section {
```

Comment:
It would be better to use a class instead of id due to saving consistency selector specificity